### ISSUE TITLE
Multi-level documentation not parsed correctly

### ISSUE THREAD

**Issue by Arcanemagus** [maxbrunsfeld/joanna#4]
Multi-level documentation is currently not being parsed correctly from JS files, for example the documentation for [`BufferedProcess::constructor`](https://atom.io/docs/api/v1.15.0/BufferedProcess#instance-constructor) shows everything at the same level, while similar documentation still in CoffeeScript files (such as [`Workspace::addModalPanel`](https://atom.io/docs/api/v1.15.0/Workspace#instance-addModalPanel)) is working correctly.

**Comment by smashwilson** [comment]
Fixed and released in 0.0.9 :sparkles:

<img width="679" alt="screen shot 2017-08-04 at 4 29 33 pm" src="https://user-images.githubusercontent.com/17565/28985800-2a612e5c-7932-11e7-863a-317b522596b5.png">

It'll make its way to atom.io Soon :tm:

**Comment by Arcanemagus** [comment]
Any idea of a timeframe on Soon ™️? 😛

Just noticed a related bug which is _likely_ already fixed, but as this isn't live I have no idea.

Currently for things with multi-level documentation all of the properties of the sub-level are listed as parameters on the main level. For example in the documentation linked above it shows the prototype for `::constructor` as this:
```
::constructor(options, command, args, options, stdout, data, stderr, data, exit, code, autoStart)
```

If this hasn't already been fixed I can file a new issue tracking it 😉.

**Comment by Arcanemagus** [comment]
Actually, I just realized that your screenshot shows it is already fixed, I just didn't notice since `::constructor(options)` is so much smaller than the giant version that is currently there, I'll go back to patiently waiting on "Soon ™️".

**Comment by smashwilson** [comment]
> Any idea of a timeframe on Soon :tm:? :stuck_out_tongue:

If we don't do anything else with it, this should go out with [1.21](https://github.com/atom/atom/blob/1.21-releases/script/package.json#L19), so not the next release but the one after that. Although it does seem fairly low-risk so maybe we could cherry-pick it into 1.20 instead :thinking:
